packages/component_code_gen/helpers/langchain_helpers.py

```diff
@@ -80,8 +80,8 @@ def create_user_prompt(prompt, urls_content):
         return prompt
 
     user_prompt = f"{prompt}\n\n## API docs\n\n"
-    for url, content in urls_content:
-        user_prompt += f"\n\n### {url}\n\n{content}"
+    for item in urls_content:
+        user_prompt += f"\n\n### {item['url']}\n\n{item['content']}"
```

Comment:
`for url, content in urls_content` was actually failing to pass docs content before — that will not iterate over the key and value of dictionary items.

Reply:
Nice! That’s probably the reason the output got worse!